models/POP/shell_scripts/cesm2_1/DART_params.csh

```diff
@@ -0,0 +1,216 @@
+
+# ==============================================================================
+# There are SourceMods that enable POP to recompute the barotropic velocity
+# to prevent the barotropic solver from crashing.
+# SourceMods may be handled in one of two ways. If you have your own git clone of
+# the repository, you may simply commit your changes to your git repo and 
+# set use_SourceMods = FALSE . If you prefer to keep your changes separate 
+# (as was required under svn), please put your SourceMods in a directory with 
+# the following structure (which is intended to be similar to the structure 
+# in the CLM distribution):
+#
+# ${SourceModDir}/src.pop
+#                |-- forcing.F90
+#                |-- initial.F90
+#                |-- overflows.F90
+#                `-- restart.F90
+
+setenv use_SourceMods TRUE
+setenv SourceModDir   ~/${cesmtag}/SourceMods
+
+# ==============================================================================
+# Directories:
+# cesmdata     Location of some supporting CESM data files.
+# cesmroot     Location of the CESM code base.
+# caseroot     Defines the CESM case directory - where the CESM+DART
+#              configuration files will be stored.  This should probably not
+#              be on a fileystem that is scrubbed.
+#              This script WILL DELETE any existing caseroot, so this script,
+#              and other useful things should be kept elsewhere.
+# rundir       Defines the location of the CESM run directory.  Will need large
+#              amounts of disk space, generally on a scratch partition.
+# exeroot      Defines the location of the CESM executable directory , where the
+#              CESM executables will be built.  Medium amount of space
+#              needed, generally on a scratch partition.
+# archdir      Defines the location of the CESM short-term archive directories.
+#              Files remain here until the long-term archiver moves them to 
+#              permanent storage.  Requires large amounts of disk space. Should
+#              not be on a scratch partition unless the long-term archiver is 
+#              invoked to move these files to permanent storage.
+
+setenv cesmdata         /glade/p/cesmdata/cseg/inputdata
+setenv cesmroot         /glade/p/cesm/releases/$cesmtag
+
+# ==============================================================================
+# Set the variables needed for the DART configuration.
+# dartroot     Location of the root of _your_ DART installation
+# BASEOBSDIR   Part of the directory name containing the observation sequence 
+#              files to be used in the assimilation. The observations are presumed
+#              to be stored in sub-directories with names built from the year and
+#              month. 'BASEOBSDIR' will be inserted into the appropriate scripts.
+# ==============================================================================
+
+setenv dartroot               /glade/work/${USER}/git/DART
```

Comment:
In all the setup scripts, etc., the variable is DARTROOT ... yet here it is lowercase.  my pmo setup bombed straightaway ... I think DARTROOT should be uppercase. I have no strong opinion on cesmdata or cesmroot or ...

Reply:
Made it DARTROOT in all of the scripts.